src/helpers/validators.js

```diff
@@ -0,0 +1,78 @@
+export const validateHelper = {
+  isEmail: value => {
+    const res = value
+      .toString()
+      .match(
+        /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+      );
+    return res !== null;
+  },
+  isUrl: value => {
+    const res = value
+      .toString()
+      .match(/(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._\+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_\+.~#?&//=]*)/g);
+    return res !== null;
+  },
+  isNumber: value => {
+    const res = value.toString().match(/\D/);
+    return res === null;
+  },
+  isEmpty: value => value.toString().trim() === "",
+  isLength: (value, len) => !!(value && value.toString().length === len),
```

Comment:
function name not clear. Use safe navigation operator operator

!! is odd to read. We can use Boolean

Reply:
`  hasEqualLength: (value, len) => Boolean(value?.toString().length === len),
`